src/main/java/com/tstepnik/planner/exceptions/TaskNotFoundException.java

```diff
@@ -0,0 +1,7 @@
+package com.tstepnik.planner.exceptions;
+
+public class TaskNotFoundException extends RuntimeException {
```

Comment:
We don't need as specific exceptions. You could've used `EntityNotFoundException` instead

Reply:
OK